src/lgar.cxx

```diff
@@ -2432,6 +2537,17 @@ extern double lgar_theta_mass_balance(int layer_num, int soil_num, double psi_cm
     if (psi_cm_loc <= 0 && psi_cm_loc_prev < 1E-50) break;
```

Comment:
`1.0E-50 is 0.0`

Reply:
I believe this is code that you included at some point; nonetheless I agree that 1e-50 is basically 0. I'm testing with 0 instead, I think that it should be fine.